pkgs/applications/video/kodi/wrapper.nix

```diff
@@ -1,8 +1,19 @@
-{ lib, makeWrapper, buildEnv, kodi, addons }:
+{ lib, makeWrapper, buildEnv, kodi, addons, callPackage }:
 
 let
+  kodiPackages = callPackage ../../../top-level/kodi-packages.nix { inherit kodi; };
+
   # linux distros are supposed to provide pillow and pycryptodome
-  requiredPythonPackages = with kodi.pythonPackages; [ pillow pycryptodome] ++ addons;
+  requiredPythonPath = kodi.pythonPackages.makePythonPath (with kodi.pythonPackages; [ pillow pycryptodome ]);
```

Comment:
```suggestion
  requiredPythonPath = with kodi.pythonPackages; makePythonPath ([ pillow pycryptodome ]);
```
Does that work?

Reply:
Yes.

```
nix-repl> kodi.pythonPackages.makePythonPath (with kodi.pythonPackages; [ pillow pycryptodome ])
"/nix/store/6cfajs6lsy9b4wxp3jvyyl1g5x2pjmpr-python3-3.8.9/lib/python3.8/site-packages:/nix/store/fs9hp5yszpykjqr172p0020m3nfraikv-python3.8-Pillow-8.2.0/lib/python3.8/site-packages:/nix/store/0sysxwcbmlknjy0938jrj33vszrg10vq-python3.8-pycryptodome-3.10.1/lib/python3.8/site-packages:/nix/store/vbka8fvf5jspiszlc7b6cwnw3k1w53gw-python3.8-olefile-0.46/lib/python3.8/site-packages"
```

Reply:
Yours looks slick. I'll use that. Thanks :+1: